client/api/activeUsers.js

```diff
@@ -0,0 +1,21 @@
+import request from './request';
+
+function getActiveUsers(id) {
+  return request.post(`/courses/${id}/get-active-users`).then(res => res.data.data);
```

Comment:
Why use ```post```, not ```get```?

You can call methods inside this api ```fetch, get & remove``` so you don't duplicate api name

Also create ```const url``` with all api paths